src/pipx/main.py

```diff
@@ -668,17 +668,8 @@ def _add_environment(subparsers: argparse._SubParsersAction) -> None:
     p = subparsers.add_parser(
         "environment",
         formatter_class=LineWrapRawTextHelpFormatter,
-        help=("Print a list of variables used in pipx.constants."),
-        description=textwrap.dedent(
-            """
-            Available variables:
-            PIPX_HOME, PIPX_BIN_DIR, PIPX_SHARED_LIBS, PIPX_LOCAL_VENVS, PIPX_LOG_DIR,
-            PIPX_TRASH_DIR, PIPX_VENV_CACHEDIR
-
-            Only PIPX_HOME and PIPX_BIN_DIR can be set by users in the above list.
-
-            """
-        ),
+        help="Print a list of environment variables and paths used by pipx",
+        description="Print a list of environment variables and paths used by pipx",
```

Comment:
```suggestion
        help="Print a list of environment variables and paths used by pipx.",
```

No point having duplicated messages here.

Reply:
I’m sticking to the original format, the message used to contain a dot.

A follow-up PR unifying the format would be nice.

Reply:
Done.